sdk/servicebus/Azure.Messaging.ServiceBus/tests/Receiver/ReceiverLiveTests.cs

```diff
@@ -28,20 +28,16 @@ await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitionin
                 await sender.SendAsync(batch);
 
                 await using var receiver = client.CreateReceiver(scope.QueueName);
-
-                Dictionary<string, string> sentMessageIdToLabel = new Dictionary<string, string>();
-                foreach (ServiceBusMessage message in sentMessages)
-                {
-                    sentMessageIdToLabel.Add(message.MessageId, Encoding.Default.GetString(message.Body.ToArray()));
-                }
+                var messageEnum = sentMessages.GetEnumerator();
 
                 var ct = 0;
```

Comment:
Why do you hate me?  Am I not worth the extra 3 characters to you?   Think of the children!   _(the ones that haven't had to deal with Hungarian notation in C++ where `ct` would be identifiable as a count)_